src/CitizenProject/Voter/AdministrateCitizenProjectVoter.php

```diff
@@ -21,7 +21,7 @@ protected function supports($attribute, $subject): bool
         return CitizenProjectPermissions::ADMINISTRATE === $attribute && $subject instanceof CitizenProject;
     }
 
-    protected function doVoteOnAttribute(string $attribute, Adherent $adherent, CitizenProject $citizenProject): bool
+    protected function doVoteOnAttribute(string $attribute, Adherent $adherent, ?CitizenProject $citizenProject): bool
     {
         if (!$citizenProject->isApproved()) {
```

Comment:
You add `?` to `$citizenProject ` so you can pass `null` to `doVoteOnAttribute()`? In this case, it will throw an exception no ?